src/features/src/changeSpeed.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import FFmpegFeature from '../FFmpegFeature';
+import Slider from '../../clui-ui-components/Slider'
+
+type ChangeSpeedConfig = {
+  SPEED: { value: number; [name: string]: any }
+  [name: string]: any
+}
+
+class ChangeSpeedFeature extends FFmpegFeature {
+  configuration: ChangeSpeedConfig
+  speed = 0;
+
+  constructor(configuration: ChangeSpeedConfig) {
+    super()
+    this.configuration = configuration
+    const { speedValue } = this.parseConfiguration()
+    this.speed = speedValue;
+    var speedStr = (1/(speedValue/100)).toString().substring(0, 4);
+    console.log(`-vf  "setpts=${speedStr}*PTS"`);
+    this.setFFmpegCommands(`-vf  \'setpts=${speedStr}*PTS\'`) 
+    // this.setFFmpegCommands('-vf  "setpts=4*PTS"') 
+    this.setProgress()
+    this.setFileConfig()
+  }
+
+  setFileConfig = () => {
+    this.fileConfig = {
+      primaryType: 'video',
+      types: [{ name: 'video', number: { min: 1, max: 1 } }]
+    }
+  }
+
+  setFFmpegCommands(command: string) {
+    this.ffmpegCommands = command
+  }
+
+  setProgress = () => {
+    if (this.speed > 1) {
+      this.progressBar.name = 'Increasing playback speed by ' + this.speed + '% ...';
+    }
+    else {
+      this.progressBar.name = 'Decreasing playback speed by ' + this.speed + '% ...';
```

Comment:
Not proper JS syntax

Reply:
Very confused by the UX here, as a User if I see decreasing speed by 1% then I am going to expect my speed to by 99%? but the UX feels like it would be 1%

Generally very confusing UX, could've simplified it a lot